fmriprep/workflows/bold/registration.py

```diff
@@ -327,35 +327,24 @@ def init_bold_t1_trans_wf(freesurfer, mem_gb, omp_nthreads, multiecho=False, use
     # Generate a reference on the target T1w space
     gen_final_ref = init_bold_reference_wf(omp_nthreads, pre_mask=True)
 
-    if not multiecho:
-        # Merge transforms placing the head motion correction last
-        nforms = 2 + int(use_fieldwarp)
-        merge_xforms = pe.Node(niu.Merge(nforms), name='merge_xforms',
-                               run_without_submitting=True, mem_gb=DEFAULT_MEMORY_MIN_GB)
-        if use_fieldwarp:
-            workflow.connect([
-                (inputnode, merge_xforms, [('fieldwarp', 'in2')])
-            ])
-
+    # Merge transforms placing the head motion correction last
+    nforms = 2 + int(use_fieldwarp)
+    merge_xforms = pe.Node(niu.Merge(nforms), name='merge_xforms',
+                           run_without_submitting=True, mem_gb=DEFAULT_MEMORY_MIN_GB)
+    if use_fieldwarp:
         workflow.connect([
-            # merge transforms
             (inputnode, merge_xforms, [
-                ('hmc_xforms', 'in%d' % nforms),
-                ('itk_bold_to_t1', 'in1')]),
-            (merge_xforms, bold_to_t1w_transform, [('out', 'transforms')]),
-            (inputnode, bold_to_t1w_transform, [('bold_split', 'input_image')]),
+                ('fieldwarp' if not multiecho else 'identity', 'in2')])
```

Comment:
@effigies is this what you meant by passing an 'identity' transform to ANTs ?

Reply:
It will need to be directly entered, not through a connection.